### DISCUSSION TITLE
各位大佬，侧边栏下方这个图标和名称部分如何在app.ts的layout设置中去掉

### DISCUSSION THREAD

**Discussion by jiazhuangle** [umijs/umi#9036] [Q&A] [answered]
![这个如何去掉](https://user-images.githubusercontent.com/2560406/185559306-6229f753-14c6-4d11-aabc-1b646f9719d8.jpg)
我在app.ts的layout设置中试着按proLayout的api都设置了一下，似乎没有找到办法，谢谢

**Comment by yangxiaoqiao2016** [comment, marked as answer]
在app.tsx中配置rightContentRender: false可去掉。但是其实是有问题的。你可以看看https://github.com/umijs/umi/issues/9060
